creation/web_base/condor_platform_select.sh

```diff
@@ -87,6 +88,41 @@ findversion_debian() {
   fi
 }
 
+findversion_os_release() {
+    local os_f=
+    local os_release dist_id dist_id_like ver major_ver
+    if [ -f "/etc/os-release" ]; then
```

Comment:
Would `[[` be better here? Or, any specific reason that single `[` was used instead?

Reply:
done